src/odemis/util/linalg.py

```diff
@@ -230,3 +230,38 @@ def are_collinear(p1: Iterable[float], p2: Iterable[float], p3: Iterable[float])
     return abs((x2 - x1) * (y3 - y1) - (x3 - x1) * (y2 - y1)) < 1e-12 and \
         abs((x2 - x1) * (z3 - z1) - (x3 - x1) * (z2 - z1)) < 1e-12 and \
         abs((y2 - y1) * (z3 - z1) - (y3 - y1) * (z2 - z1)) < 1e-12
+
+
+def generate_triangulation_points(max_dist: float, area_coords: Iterable[float]) -> Iterable[Tuple[float]]:
+    """
+    Finds (x,y) positions on the given area. The points are evenly distributed from each other in the given area.
+    The distance between points is decided by maximum distance.
+
+    :param max_dist: the maximum distance allowed between two (x,y) positions.
+    :param area_coords: [xmin, ymin, xmax, ymax] the top right and bottom left (x,y) coordinates in meters.
+    :return: List of (x,y) coordinates in the given area.
+    """
+    xmin, ymin, xmax, ymax = area_coords
+
+    # Avoid points exactly on the border of the given area, find points delta distance
+    # away from the border of the given area
+    length_x = abs(xmax - xmin)
+    length_y = abs(ymax - ymin)
+    points_x = numpy.floor(length_x/max_dist) + 1
+    points_y = numpy.floor(length_y/max_dist) + 1
+    xmin = xmin + max_dist / 2
```

Comment:
You are now using a max distance of 450µm, I think a tile in the overview image for meteor is around 80µm. This means that your first focus point is 3 tiles away from the border, is that not a bit much?

Reply:
450µm is the maximum distance between two adjacent focus points. With the overview acquisition hardware testing, the focus points separated by this distance could provide focused image. This idea was then extended to set distance from the border. If x is the max_distance between two focus points then x/2 would be the focus range of a single focus point.